tests/Feature/WebRouteTest.php

```diff
@@ -81,6 +81,36 @@ public function test_results_route()
             });
     }
 
+    /**
+     * Test that the /results response contains mismatch data
+     *
+     *  @return void
+     */
+    public function test_results_route_retrieves_mismatches()
+    {
+        $import = ImportMeta::factory()
+        ->for(User::factory()->uploader())
+        ->create();
+
+        $mismatch = Mismatch::factory()
+            ->for($import)
+            ->create();
+
+        $response = $this->get(route('results', [
+            'ids' => $mismatch->item_id
+        ]));
+
+        $response->assertSuccessful();
+        $response->assertViewIs('app')
+            ->assertInertia(function (Assert $page) use ($mismatch, $import) {
+                $page->component('Results')
+                    ->has('results.0', 11) // result object size
```

Comment:
Non-blocking, but I'm not sure if this is needed. We care that it shows particular parts of the data, if we tie this to the size of a mismatch object it will make the test more brittle.

Reply:
Yes, this could change. I was also not sure about using the number.

Reply:
see the discussion we had on the size test in the other PR: https://github.com/wmde/wikidata-mismatch-finder/pull/109#pullrequestreview-758480808